sdk/python/feast/feature_store.py

```diff
@@ -602,11 +602,18 @@ def _make_inferences(
 
         # New feature views may reference previously applied entities.
         entities = self._list_entities()
+        provider = self._get_provider()
         update_feature_views_with_inferred_features_and_entities(
-            views_to_update, entities + entities_to_update, self.config
+            provider,
+            views_to_update,
+            entities + entities_to_update,
+            self.config,
         )
         update_feature_views_with_inferred_features_and_entities(
-            sfvs_to_update, entities + entities_to_update, self.config
+            provider,
```

Comment:
There should be a new one for ODFVs

Reply:
yep, 3 weeks ago it was not there 😉 
will add it